docs/data-structures.rst

```diff
@@ -97,29 +98,14 @@ Transaction Test (object)
 .. _transaction-results:
 
 Transaction Results (object)
-----------------------------
-
-This is a cousin of the :ref:`gavel-validation-result`.
-
--  general (object) - contains Dredd’s custom messages (e.g. “test was skipped”), formatted the same way like those from Gavel
-
-   -  results (array[:ref:`gavel-error`])
-
--  statusCode (:ref:`gavel-validator-output`)
--  headers (:ref:`gavel-validator-output`)
--  body (:ref:`gavel-validator-output`)
-
-.. _gavel-validation-result:
-
-Gavel Validation Result (object)
 --------------------------------
 
-Can be seen also `here <https://relishapp.com/apiary/gavel/docs/javascript/request-validation#validate>`__ and `here <https://relishapp.com/apiary/gavel/docs/javascript/response-validation#validate>`__.
+Transaction result equals to the validation result of the Gavel validation result. You can read more about the `Gavel validation structure <https://github.com/apiaryio/gavel.js>`__.
+
+-  valid (boolean) - Indicates whether a transaction is valid.
+-  fields (object)
 
--  statusCode (:ref:`gavel-validator-output`)
--  headers (:ref:`gavel-validator-output`)
--  body (:ref:`gavel-validator-output`)
--  version (string) - version number of the Gavel Validation Result structure
+   - [fieldName: string]: :ref:`gavel-validator-output`
```

Comment:
In MSON this would be written as

```
- fields (object)
  - *body* - :ref:`gavel-validator-output`
```

but since this isn't entirely MSON anymore, do as you like. This needs to be fixed once with https://github.com/apiaryio/dredd/issues/1134

Reply:
It might be nice to document what are the possible or at least expected values for `fieldName`.